### PR TITLE
fix non-escaped chars

### PR DESCRIPTION
**Pull request by altush1983**

### CODE REVIEW

File: WinImageBuilder.psm1 [hunk 1]
```diff
@@ -1155,9 +1155,7 @@ function New-WindowsOnlineImage {
             throw "Selected vmswitch {0} does not exist" -f $windowsImageConfig.external_switch
         }
         if ($switch.SwitchType -ne "External" -and !$windowsImageConfig.force) {
-            throw "Selected switch {0}} is not an external
-                switch. If you really want to continue use the `force = true` flag." `
-                -f $windowsImageConfig.external_switch
+            throw ("Selected switch {0} is not an external switch. If you really want to continue use the ``force = true`` flag." -f $windowsImageConfig.external_switch)
```
Comment (ader1990) [inline WinImageBuilder.psm1]: The line is too long, can you change it to the original size?

File: WinImageBuilder.psm1 [hunk 2]
```diff
@@ -1389,9 +1387,7 @@ function New-WindowsFromGoldenImage {
             throw "Selected vmswitch {0} does not exist" -f $windowsImageConfig.external_switch
         }
         if ($switch.SwitchType -ne "External" -and !$windowsImageConfig.force) {
-            throw "Selected switch {0} is not an external
-                switch. If you really want to continue use the `force = true` flag." `
-                -f $windowsImageConfig.external_switch
+            throw ("Selected switch {0} is not an external switch. If you really want to continue use the ``force = true`` flag." -f $windowsImageConfig.external_switch)
```
Comment (ader1990) [inline WinImageBuilder.psm1]: ditto

**Comment by ader1990** [comment]
@altush1983 can you please address the review comments? The change looks good and will be merged after you address the comments.

**Comment by altush1983** [comment]
I could not split the code line into several. Current edit is ok

**Pull request merged**